tests/fixtures/mock_decision.py

```diff
@@ -10,7 +10,6 @@
   "contents": [
     {
       "type": "html",
-      "body": "Example",
```

Comment:
Did we ever get an "Example" value here? I think it's safe to assume we will only get valid json in the body from now on.

Reply:
If body is a manual entry, I'm hesitant to make this assumption. Maybe we should deal with possible exceptions thrown during `json.loads` (just make body = None and let the topic method deal with it).

Reply:
It's not a manual entry. The AdZerk ui has checkboxes that get json-encoded in the Decision API.